grafana/src/eaRelease.jsonnet

```diff
@@ -0,0 +1,276 @@
+// https://grafana.com/docs/grafana/latest/dashboards/json-model/
+local grafana = import 'grafonnet/grafana.libsonnet';
+local graphPanel = grafana.graphPanel;
+local timeseries = grafana.timeseries;
+local textPanel = grafana.text;
+local prometheus = grafana.prometheus;
+
+local shared = import './shared.libsonnet';
+local addSideLegend = shared.helpers.addSideLegend;
+
+local cortexDataSource = shared.constants.cortexDataSource;
+local instanceFilter = shared.constants.instanceFilter;
```

Comment:
This instance filter also adds a filter for service, which unfortunately won't work for all queries in the dashboard (nginx vs ea metrics)

Reply:
I created a new filter namespaceFilter that only filters namespace and leaves the service unfiltered and replaced all instances of instanceFilter with namespaceFilter in the code, Does that resolve this issue?

Reply:
The adapter label in instanceFilter will clash with the service filter it already has, better to leave that one alone. Idk also 
 if that service label is coming from k8s, I think so

Reply:
What is this service filter doing? I don't see how it effects anything?